### DISCUSSION TITLE
Issue Management

### DISCUSSION THREAD

**Discussion by shivarm** [Hyperfoil/Horreum#614] [Q&A] [answered]
@jesperpedersen look at this-https://github.com/quarkusio/quarkus/blob/dcc17a6887dcb375bba43fd4981cfce8ff551572/pom.xml#L48C5-L48C22
can we do same for our project and would you like to explain what is does?

**Comment by jesperpedersen** [comment, marked as answer]
Yeah, we can do the same thing - our `pom.xml` and Java `MANIFEST.MF` files should be up-to-date

**Reply by shivarm** [reply]
> `MANIFEST.MF` files should be up-to-date

any idea?

**Reply by shivarm** [reply]
@jesperpedersen looking for this file but don't know, where is it locate?

**Reply by jesperpedersen** [reply]
See a related commit - https://github.com/Hyperfoil/Hyperfoil/commit/a02126eb29707c2569f00d58543fd166bf642f7b

**Reply by shivarm** [reply]
@jesperpedersen here I did same- https://github.com/quarkiverse/quarkus-quinoa/pull/462#event-10022391078 and I did't have to do with `MANIFEST.MF`

**Reply by jesperpedersen** [reply]
We don't depend on `jboss-parent` - verify the `MANIFEST.MF` inside the artifacts to make sure
